test/integration/bundles.js

```diff
@@ -47,9 +46,27 @@ describe.skip('Bundles - Integrations', () => {
 
     scenario = new ScenarioBuilder(apparatus.identityManager, apparatusScenarioProcessor(apparatus));
     await scenario.addAdminAccount(adminAccountWithSecret);
-
-    const from = getDefaultAddress(await createWeb3());
-    await apparatus.bundleProofRegistryContract.methods.addToWhitelist(from, url).send({from});
+    const from = apparatus.identityManager.nodeAddress();
+
+    const kycContract = await apparatus.contractManager.kycWhitelistContract();
+    const HERMES_ROLE_INDEX = 2;
+    await kycContract
+      .methods
+      .add(from, HERMES_ROLE_INDEX)
+      .send({
+        from,
+        gas: 2000000
+      });
+
+
+    const rolesContract = await apparatus.contractManager.rolesContract();
+    await rolesContract
+      .methods
+      .onboardAsHermes(url)
+      .send({
+        from,
+        gas: 2000000
+      });
```

Comment:
Can we use repositories for this stuff?

Reply:
I'd use here prerun utils from Your PR.
Will change it after merging Yours.

Reply:
https://english.stackexchange.com/questions/30185/you-versus-you-as-polite-form-of-writing